tests/unit_tests/configuration/test_configurator.py

```diff
@@ -181,3 +181,21 @@ def test_get_db_parameters(configurator, args_dict):
     args_dict["db_api_authentication_database"] = "admin"
 
     assert configurator.config == args_dict
+
+
+def test_initialize_output(configurator, args_dict):
+
+    configurator.parser.initialize_output_arguments()
+    configurator._fill_from_command_line(arg_list=[])
+
+    print(configurator.config)
```

Comment:
This is probably left from debugging and shouldn't be here.

Reply:
Yes. Removed it.